runtime/pom.xml

```diff
@@ -47,4 +41,17 @@
       </plugin>
     </plugins>
   </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-arc</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>net.sf.jasperreports</groupId>
+      <artifactId>jasperreports</artifactId>
+      <version>${jasperreports.version}</version>
+    </dependency>
```

Comment:
If Jasper includes libs like POI you might want to do an Exclude here and use Quarkus POI which has already been updated for GraalVM Native.  There may be some others as well.

Reply:
done, at least in integration tests. For xlsx files, poi needs to be provided by the user.